Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["src/firecracker", "src/jailer", "src/seccompiler", "src/rebase-snap"]
+members = ["src/firecracker", "src/jailer", "src/seccompiler", "src/rebase-snap", "src/guest_config"]
```

Comment:
I think this does not need to be here, since it's a dependency? These are only for top-level binaries AFAIR.

Reply:
I will look into this a bit more. It's possible I missed something when configuring the crate, cause clippy required this entry, but I do agree that it shouldn't be necessary.